test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -43,13 +43,15 @@ call_log_manager::call_log_manager(const std::string &call_log_file)
 
     _call_log = json::parse(file);
     api_map_setup();
+    _conn = nullptr;
```

Comment:
@Sean04 I have assigned conn here, to make sure everything is assigned in one place, i.e. constructor.

Reply:
You can use the [constructor initializer list](https://en.cppreference.com/w/cpp/language/constructor) for this.
```suggestion
    call_log_manager::call_log_manager(const std::string &call_log_file): _conn(nullptr)
```

Reply:
Thanks for this suggestion, I like the idea of using an initializer list. Updated in [1ccbacb](https://github.com/wiredtiger/wiredtiger/pull/8118/commits/1ccbacb59d7a880c240f5db27ba5676236bafd5c)